packages/client-sdk-nodejs/src/index.ts

```diff
@@ -117,6 +117,7 @@ import {
   CredentialProvider,
   StringMomentoTokenProvider,
   EnvMomentoTokenProvider,
+  getDefaultCredentialProvider,
```

Comment:
Do we intend for this to be a user-facing function? If not, I think we might not want to export it from the nodejs/web sdk packages.

Reply:
oh probably not, will remove it from there